src/components/Header.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+const Header: React.FC = () => {
+	return <h1>Test</h1>
+}
```

Comment:
This can be shorten. :stuck_out_tongue_winking_eye:
```suggestion
const Header: React.FC = () => <h1>Test</h1>
```

Reply:
We already talked about this. Writing about this in english again won't help anyone.

Reply:
Header component was created as an example of an component we will base all other components on. We, quite desperately, NEED it to show how it's done, not how it can be done on a senior level with shortening things we can hardly process, let alone write on our own. 

In other words, shortening this without any other component to look on for the correct way to write them will just mean multiple people will go, create a component of their own... and get stuck there because it won't work without correct brackets and when we'll look on our example component (Header), they are not there.